packages/wallet/src/utils/json-rpc-utils.ts

```diff
@@ -65,3 +71,22 @@ export function createStateFromCreateChannelParams(params: JsonRpcCreateChannelP
     isFinal: false
   };
 }
+
+// TODO: Error handling
+export function createStateFromUpdateChannelParams(
+  state: State,
+  params: JsonRpcUpdateChannelParams
+): State {
+  const {appData, allocations} = params;
+
+  // TODO: check for valid transition using EVM library
```

Comment:
I don't think we need to check for a valid transition in `createStateFromUpdateChannelParams` so we can probably remove this `TODO`